lib/packwerk/application_load_paths.rb

```diff
@@ -48,7 +48,9 @@ def filter_relevant_paths(all_paths, bundle_path: Bundler.bundle_path, rails_roo
 
       sig { params(paths: T::Array[Pathname], rails_root: Pathname).returns(T::Array[String]) }
       def relative_path_strings(paths, rails_root: Rails.root)
-        paths.map { |path| path.relative_path_from(rails_root).to_s }
+        paths
+          .map { |path| path.relative_path_from(rails_root).to_s }
+          .uniq
```

Comment:
We need this `.uniq` because `#filter_relevant_paths` may result in some added load paths. This was causing a failure when I tophatted on Core.

Reply:
The `.uniq` was removed from `ApplicationValidator`, but I figure it is best to add it in this class so we can also test for it.